ca.vlacroix.Tally.json

```
@@ -0,0 +1,38 @@
+{
+	"app-id": "ca.vlacroix.Tally",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "47",
+	"sdk": "org.gnome.Sdk",
+	"command": "tally",
+	"finish-args": [
+		"--device=dri",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [
+		"shared-modules/lua5.4/lua-5.4.json",
+		{
+			"name": "lgi",
+			"no-autogen": true,
+			"make-args": ["-C", "lgi", "LUA_VERSION=5.4"],
+			"make-install-args": ["-C", "lgi", "LUA_VERSION=5.4", "PREFIX=/app"],
+			"sources": [{
+				"type": "archive",
+				"url": "https://github.com/lgi-devs/lgi/archive/e06ad94c8a1c84e3cdb80cee293450a280dfcbc7.zip",
+				"sha256": "003984a7a33236cc2368bf8c87905b6867d1292f844df236051a7b45f134d272"
+			}]
+		},
+		{
+			"name": "tally",
+			"no-autogen": true,
+			"make-args": [],
+			"make-install-args": [],
```

```suggestion
			"make-args": [],
			"make-install-args": [],
```

You can remove these two if it works with the default.